core/neat/mixins/_grid-container.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+
+/// Creates Neat grid container with clearfix.
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-container;
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: auto;
+///     margin-right: auto;
+///   }
+///
+///   .element::after {
+///     clear: both;
+///     content: "";
+///     display: block;
+///   }
+
+@mixin grid-container($grid: $neat-default-grid) {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Yes, we had chatted a bit about that earlier and defaulted to this. But maybe the best thing to do is not try to do anything, and let the developer do it. 

Other option is to handle it in the `$grid` map definition, `center`, `left`, or `right` but that seems like a lot of logic for little benefit. Also, on more than one project we wanted it left aligned at large breakpoints, and centered at lower ones.